notebooks/file_calibration.ipynb

```diff
@@ -61,6 +61,7 @@
     "from hera_notebook_templates.data import DATA_PATH as HNBT_DATA\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.            flags = vissol_flags.get((bl[0], bl[1], 'nn'), rfi_flags) | vissol_flags.get((bl[0], bl[1], 'ee'), rfi_flags)
</pre><p>I don't think you need to use .get here.</p><p> </p><p>Also, I wonder whether you're overwriting <code>flags</code> somewhere... </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
There were "missing" keys in vissol_flags, but it just turned out to be baselines with just one cross-pol flagged. Fixed this to just use `omni_flags` which should always have the right keys